ktlint-core/src/test/kotlin/com/pinterest/ktlint/core/internal/EditorConfigGeneratorTest.kt

```diff
@@ -0,0 +1,138 @@
+package com.pinterest.ktlint.core.internal
+
+import com.google.common.jimfs.Configuration
+import com.google.common.jimfs.Jimfs
+import com.pinterest.ktlint.core.Rule
+import com.pinterest.ktlint.core.RuleSet
+import com.pinterest.ktlint.core.api.FeatureInAlphaState
+import com.pinterest.ktlint.core.api.UsesEditorConfigProperties
+import java.nio.file.FileSystem
+import java.nio.file.Files
+import java.nio.file.Path
+import org.assertj.core.api.Assertions.assertThat
+import org.jetbrains.kotlin.com.intellij.lang.ASTNode
+import org.junit.Test
+
+@OptIn(FeatureInAlphaState::class)
+internal class EditorConfigGeneratorTest {
```

Comment:
This will be probably also failing on windows because the path separators should be replaced with `File.separator` I think

Reply:
It was suggested a while back we start running CI tests on Azure as well but I haven't yet looked in to it.

Reply:
I think we wanted to move to github actions instead where we could also run tests on windows

Reply:
`EditorConfigLoader` already supports loading on Windows fs. This class does not uses passed `filePath` to output anything.